### PR TITLE
fix(annotations): fix and reactivate integration tests

### PR DESCRIPTION
**Pull request by peetcremer1**
This reactivates the Annotations integration tests that have been deactivated in [this PR ](https://github.com/cognitedata/cognite-sdk-js/pull/770)
and fixes them by making them less prone to fail due to detail changes on the API.

In particular, they won't fail anymore when optional fields are added to Annotation Payloads on the API side.

### CODE REVIEW

**Review by henrist — APPROVED** [review]

File: packages/playground/src/__tests__/api/annotationsApi.int.spec.ts [hunk 1]
```diff
@@ -111,7 +109,14 @@ describe.skip('Annotations API', () => {
     expect(annotation.creatingApp).toEqual(partial.creatingApp);
     expect(annotation.creatingAppVersion).toEqual(partial.creatingAppVersion);
     expect(annotation.status).toEqual(partial.status);
-    expect(annotation.data).toEqual(partial.data);
+    expect(annotation).toHaveProperty('data');
+    const annotation_data: any = annotation.data;
```
Comment (cyriaquebrousse) [inline packages/playground/src/__tests__/api/annotationsApi.int.spec.ts]:
```suggestion
    const annotationData: any = annotation.data;
```
🐫

File: packages/playground/src/__tests__/api/annotationsApi.int.spec.ts [hunk 2]
```diff
@@ -178,26 +183,30 @@ describe.skip('Annotations API', () => {
...
     ];
     const updatedResp = await client.annotations.update(changes);
     const updated = updatedResp[0];
-    expect(updated.data).toEqual({
-      pageNumber: 8,
-      fileRef: { externalId: 'def_file_changed', id: null },
-      textRegion: { xMin: 0, xMax: 0.1, yMin: 0, yMax: 0.2 },
-    });
+    expect(updated).toHaveProperty('data');
+    const updated_data: any = updated.data;
```
Comment (cyriaquebrousse) [inline packages/playground/src/__tests__/api/annotationsApi.int.spec.ts]: same

**Review by cyriaquebrousse — APPROVED** [review]
Minor syntax comments but looks good

File: packages/playground/src/__tests__/api/annotationsApi.int.spec.ts (hunk 1 above)
Comment (cognite-ornellas) [reply packages/playground/src/__tests__/api/annotationsApi.int.spec.ts]: I think that makes sense to change this variable's name to camelCase name. Can you change it?

File: packages/playground/src/__tests__/api/annotationsApi.int.spec.ts (hunk 2 above)
Comment (cognite-ornellas) [reply packages/playground/src/__tests__/api/annotationsApi.int.spec.ts]: I think that makes sense to change this variable's name to camelCase name. Can you change it?

File: packages/playground/src/__tests__/api/annotationsApi.int.spec.ts (hunk 1 above)
Comment (peetcremer1) [reply packages/playground/src/__tests__/api/annotationsApi.int.spec.ts]: Done here: https://github.com/cognitedata/cognite-sdk-js/pull/771/commits/59592f87a2f7e43c0867d9409ae8cfe14f1174bb

File: packages/playground/src/__tests__/api/annotationsApi.int.spec.ts (hunk 2 above)
Comment (peetcremer1) [reply packages/playground/src/__tests__/api/annotationsApi.int.spec.ts]: Done here https://github.com/cognitedata/cognite-sdk-js/pull/771/commits/59592f87a2f7e43c0867d9409ae8cfe14f1174bb

**Review by cognite-ornellas — APPROVED** [review]

**Pull request merged**